search-app/docs/api.md

```diff
@@ -3686,15 +3686,16 @@ On occasions when variable association cannot be processed at all due to invalid
 
 #### <a name="variable-dissociation"></a> Variable Dissociation
 
-A variable identified by its concept id can be dissociated from collections through a list of collection concept revisions similar to variable association requests.
+A variable identified by its concept id can be dissociated from collections similar to variable association requests.
+Future work:
```

Comment:
It becomes apparent that we need to change the variable dissociation route once you put this in the API doc. We should just change the dissociation route to something like:
```
DELETE %CMR-ENDPOINT%/associate/variables/V1200000008-PROV1/collections/C1200000002-PROV1
```
rather than changing this awkward dissociation endpoint that is for multiple concepts which has been eliminated from our use cases. This endpoint will be in-sync with the future change to make variable association route to the same route with POST. So I think we should add the above dissociation endpoint and mark this existing endpoint as deprecated. We don't need to change the existing route to only accept a single collection concept.
Sorry that I didn't come to this realization immediately when I first reviewed the PR.